src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Sort all persons in the address book and list to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book sorted by names.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        List<ReadOnlyPerson> sortedPersons = new ArrayList<ReadOnlyPerson>();
+        int i,j;
+        for(i=0;i<allPersons.size();i++){
```

Comment:
Spacing required here. Refer to coding standard for better coding style in Java.